components/bsx/Gallery/Item/GalleryItem.vue

```diff
@@ -407,24 +409,24 @@ export default class GalleryItem extends mixins(
     return !isShareMode
   }
 
-  @Watch('meta', { deep: true })
-  handleNFTPopulationFinished(newVal) {
-    if (newVal) {
-      // save visited detail page to history
-      this.$store.dispatch('history/addHistoryItem', {
-        id: this.id,
-        name: this.nft.name,
-        image: this.meta.image,
-        collection: (this.nft.collection as any).name,
-        date: new Date(),
-        description: this.meta.description,
-        author: this.nft.currentOwner,
-        price: this.nft.price,
-        mimeType: this.mimeType,
-        prefix: this.urlPrefix,
-      })
-    }
-  }
+  // @Watch('meta', { deep: true })
+  // handleNFTPopulationFinished(newVal) {
+  //   if (newVal) {
+  //     // save visited detail page to history
+  //     this.$store.dispatch('history/addHistoryItem', {
+  //       id: this.id,
+  //       name: this.nft.name,
+  //       image: this.meta.image,
+  //       collection: (this.nft.collection as any).name,
+  //       date: new Date(),
+  //       description: this.meta.description,
+  //       author: this.nft.currentOwner,
+  //       price: this.nft.price,
+  //       mimeType: this.mimeType,
+  //       prefix: this.urlPrefix,
+  //     })
+  //   }
+  // }
```

Comment:
Why we delete this part?
IMO it's being used in somewhere at BSX currently.

Reply:
Because it is crashing on the BSX :)

Reply:
5b955bab7425d21c7c9dc7b6bf6399d5f6548514

Reply:
This part is related to the History component.

<img width="1580" alt="image" src="https://user-images.githubusercontent.com/31397967/180967766-70b44877-36ba-4220-8d41-031400bca399.png">

Reply:
> Because it is crashing on the BSX :)

 Aright, maybe we could create another issue to trace it : )